### PR TITLE
[slashnext-251] Added SlashNext Playbooks and Updated as per Pack str…

### PR DESCRIPTION
**Pull request by SaadatAbid**
…ucture update.

Signed-off-by: Saadat Abid <saadatabid@gmail.com>

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by dantavori** [comment]
@SaadatAbid please register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA) before we proceed with the review.

**Comment by SaadatAbid** [comment]
@dantavori Can you help me make an offline zip for SlashNext pack so that we can our internal QA can verify changes as a whole?

**Comment by dantavori** [comment]
Hey @SaadatAbid,
1. It is possible to get a zip file using `demisto-sdk create-content-artifacts`, but before you do it you need to change a specific server configuration and after you download the file it needs to be deleted immediately (for security reasons). I can give you instructions or if you prefer, we can create it for you.
2. For contributing a premium pack, follow the instructions in our contribution guide - see section 3 [here](https://xsoar.pan.dev/docs/contributing/contributing#how-to-contribute).
3. Let's talk on Slack to schedule a demo. Sending you a message on our DFIR workspace.

**Comment by SaadatAbid** [comment]
> Hey @SaadatAbid,
> 
>     1. It is possible to get a zip file using `demisto-sdk create-content-artifacts`, but before you do it you need to change a specific server configuration and after you download the file it needs to be deleted immediately (for security reasons). I can give you instructions or if you prefer, we can create it for you.
> 
>     2. For contributing a premium pack, follow the instructions in our contribution guide - see section 3 [here](https://xsoar.pan.dev/docs/contributing/contributing#how-to-contribute). Contact me if you need more details.
> 
>     3. Let's talk on Slack to schedule a demo. Sending you a message on our DFIR workspace.

Hey @dantavori,
1. I understand. We will delete it immediately. Please give me the instructions so that I can build the package now and with any change if that is required in future. 
2. Ok I have already created three different packages using the above instructions and invited `xsoar-dbot` to collaborate. 
3. Sure. Just FYI we already had a demo with XSOAR team for the playbooks and incorporated their feedback in this PR. Earlier I had created PR using marketplace contribution so I was not able to edit it that's why I am including those changes here.
For reference these were the earlier PRs; [9147](https://github.com/demisto/content/pull/9147) and [9161](https://github.com/demisto/content/pull/9161)

**Review by dantavori — CHANGES_REQUESTED** [review]
1. In `Online_Brand_Protection_Detect_and_Respond` when no URLs are found after Extract URL, you don't close the incident (you did close in the previous version) - is that on purpose?
2. Please generate docs for the playbook.

**Comment by SaadatAbid** [comment]
>     1. In `Online_Brand_Protection_Detect_and_Respond` when no URLs are found after Extract URL, you don't close the incident (you did close in the previous version) - is that on purpose?
> 
>     2. Please generate docs for the playbook.

1. I don't really know how it happened but it's not intentional. Let me update it.
2. Can you guide me in this regard? Do I have to do this manually or is there any script for that?

File: Packs/SlashNextPhishingIncidentResponse/ReleaseNotes/1_2_0.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+  - When combined with ‘SlashNext Abuse Management Protection’, this playbook fully automates the identification and remediation of phishing emails found in Microsoft 365 user inboxes. Using the indicators of compromise, URL, domain, and IP, found in the original email, it searches and remediates other emails containing the same IOCs.
+##### New: Online Brand Protection Detect and Respond
+  - Analyzes the domains and URLs in suspicious emails, reported by end users, to determine if the phishing campaign is impersonating your company’s brand. Playbook can then trigger a domain take down email, with forensic evidence, to a target address.
+##### New: Abuse Inbox Management Protection
+  - Analyzes the URLs, domains, and IPs in suspicious emails, reported by end users, and returns a binary verdict (malicious or benign) and forensic information including screenshot of attack page, threat name and type, threat status, and first/last seen date
+#### Scripts
+##### New: BrandImpersonationDetection
+  - Analyzes the forensic data to detect brand impersonation attacks. Modify this file with the attributes associated with your company’s brand.
```
Comment (dantavori) [inline Packs/SlashNextPhishingIncidentResponse/ReleaseNotes/1_2_0.md:10]:
```suggestion
  - When combined with ***SlashNext Abuse Management Protection***, this playbook fully automates the identification and remediation of phishing emails found in Microsoft 365 user inboxes. Using the indicators of compromise, URL, domain, and IP, found in the original email, it searches and remediates other emails containing the same IOCs.
##### New: Online Brand Protection Detect and Respond
  - Analyzes the domains and URLs in suspicious emails, reported by end users, to determine if the phishing campaign is impersonating your company’s brand. Playbook can then trigger a domain take down email, with forensic evidence, to a target address.
##### New: Abuse Inbox Management Protection
  - Analyzes the URLs, domains, and IPs in suspicious emails, reported by end users, and returns a binary verdict (malicious or benign) and forensic information including screenshot of attack page, threat name and type, threat status, and first/last seen date.
#### Scripts
##### New: BrandImpersonationDetection
  - Analyzes the forensic data to detect brand impersonation attacks. Modify this file with the attributes associated with your company’s brand.
```

**Review by dantavori — APPROVED** [review]
great work!

**Pull request merged**